These stdlib functions are required in Handlers.cpp but not present.

It won't compile for osx-arm64 metal; unsure why it can on linux-64, but in any
case, they should be present.
Index: pytorch/torch/csrc/distributed/c10d/control_plane/Handlers.hpp
===================================================================
--- pytorch.orig/torch/csrc/distributed/c10d/control_plane/Handlers.hpp	2024-11-12 20:18:55.762951714 -0600
+++ pytorch/torch/csrc/distributed/c10d/control_plane/Handlers.hpp	2024-11-19 16:23:52.134859846 -0600
@@ -4,6 +4,8 @@
 #include <map>
 #include <string>
 #include <utility>
+#include <unordered_map>
+#include <vector>
 
 #include <c10/macros/Export.h>
 
